skyportal/handlers/api/source.py

```diff
@@ -77,1919 +77,2067 @@
+              type: string
             description: |
-              Boolean indicating whether to include photometry detection statistics for each source
-              (last detection and peak detection). Defaults to false.
+              Arrow-parseable date string (e.g. 2020-01-01). If provided, filter by
+              PhotStat.last_detected_mjd <= endDate
           - in: query
-            name: classifications
+            name: listName
             nullable: true
             schema:
-              type: array
-              items:
-                type: string
-            explode: false
-            style: simple
+              type: string
             description: |
-              Comma-separated string of "taxonomy: classification" pair(s) to filter for sources matching
-              that/those classification(s), i.e. "Sitewide Taxonomy: Type II, Sitewide Taxonomy: AGN"
+              Get only sources saved to the querying user's list, e.g., "favorites".
           - in: query
-            name: nonclassifications
+            name: group_ids
             nullable: true
             schema:
-              type: array
+              type: list
               items:
-                type: string
-            explode: false
-            style: simple
+                type: integer
             description: |
-              Comma-separated string of "taxonomy: classification" pair(s) to filter for sources NOT matching
-              that/those classification(s), i.e. "Sitewide Taxonomy: Type II, Sitewide Taxonomy: AGN"
+               If provided, filter only sources saved to one of these group IDs.
           - in: query
-            name: annotationsFilter
+            name: includePhotometry
             nullable: true
             schema:
-              type: array
-              items:
-                type: string
-            explode: false
-            style: simple
+              type: boolean
             description: |
-              Comma-separated string of "annotation: value: operator" triplet(s) to filter for sources matching
-              that/those annotation(s), i.e. "redshift: 0.5: lt"
+              Boolean indicating whether to include associated photometry. Defaults to
+              false.
           - in: query
-            name: annotationsFilterOrigin
+            name: includeColorMagnitude
             nullable: true
             schema:
-              type: string
-            description: Comma separated string of origins. Only annotations from these origins are used when filtering with the annotationsFilter.
+              type: boolean
+            description: |
+              Boolean indicating whether to include the color-magnitude data from Gaia.
+              This will only include data for objects that have an annotation
+              with the appropriate format: a key named Gaia that contains a dictionary
+              with keys named Mag_G, Mag_Bp, Mag_Rp, and Plx
```

Comment:
I think this format has changed a bit when we added new catalogs.

Reply:
yup!